Display/features/calculating_tubing_descriptors.feature

```diff
@@ -0,0 +1,11 @@
+Feature: Calculating tubing descriptors
+
+    Scenario: Calculating descriptors
+        Given we can use sensor data
```

Comment:
These kinds of tests are generally something like "Given we have this set of objects, when this data is given to the objects, then these results happen".  You've combined steps here.  As the test suite expands, you may want the steps to be separated so that you can reuse steps.  For instance, if you are always instantiating the same set of objects and storing them in the context, then that set can be in the givens.  _When_ you calculate the descriptors, _then_ the results of a particular set of input data match your expectations.  Right now, this reads that you're calculating things, but you're not checking the output of the calculations for correctness.